msg/differential_pressure.msg

```diff
@@ -1,6 +1,10 @@
-uint64 timestamp				# time since system start (microseconds)
-uint64 error_count				# Number of errors detected by driver
-float32 differential_pressure_raw_pa		# Raw differential pressure reading (may be negative)
-float32 differential_pressure_filtered_pa	# Low pass filtered differential pressure reading
-float32 temperature				# Temperature provided by sensor, -1000.0f if unknown
-uint32 device_id				# unique device ID for the sensor that does not change between power cycles
+uint64 timestamp                     # time since system start (microseconds)
+uint64 timestamp_sample
+
+uint32 device_id                     # unique device ID for the sensor that does not change between power cycles
+
+float32 differential_pressure_pa     # differential pressure reading in Pascals (may be negative)
+
+float32 temperature                  # Temperature provided by sensor, -1000.0f if unknown
```

Comment:
>-1000.0f if unknown

This comment needs to be updated, as we now use NaN in that case.

Reply:
```suggestion
float32 temperature_celsius                  # Temperature provided by sensor, -1000.0f if unknown
```

Reply:
Comment fixed.

For the temperature field I think we should decide if what to do in general and then do it consistently everywhere. At the moment all the other sensor messages just have "temperature".

Reply:
right, certainly not that as important in the context of this PR. 
But yeah I'd vote for putting the unit in the variables consistently.